chia/harvester/harvester_api.py

```diff
@@ -67,6 +70,9 @@ async def new_signage_point_harvester(
             # This means that we have not received the handshake yet
             return None
 
+        config = load_config(DEFAULT_ROOT_PATH, "config.yaml")
```

Comment:
We don't want to load the config on every run. Use the harvester config: `self.harvester.config`, or even better just print out the puzzle hash. The harvester knows nothing about address encodings